modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -363,7 +339,6 @@ public void processOpts() {
             importMapping.put("LocalDate", "org.joda.time.LocalDate");
             importMapping.put("DateTime", "org.joda.time.DateTime");
         } else if (dateLibrary.startsWith("java8")) {
-            additionalProperties.put("java8", "true");
```

Comment:
If we remove this, all the {{#java8}} mustache tabs in the template will become false, right? 

I still find reference to {{#java8}} in the templates:
```
swagger-codegen|remove_format⚡ ⇒ grep -R '{{#java8}}' modules/swagger-codegen/src/main/resources/Java/* | wc -l
      37
```